documentation/modules/security/con-securing-client-authentication.adoc

```diff
@@ -6,8 +6,9 @@
 = User authentication
 
 [role="_abstract"]
-User authentication is configured using the `authentication` property in `KafkaUser.spec`.
-The authentication mechanism enabled for the user is specified using the `type` field.
+Use the `KafkaUser` custom resource to configure an authentication mechanism for clients that require access to a Kafka cluster. 
+Configure user authentication using the `authentication` property in `KafkaUser.spec`.
+Enable an authentication mechanism using the `type` field.
```

Comment:
Same as above, this does not enable the mechanism. It just controls the type of credentials which will be generated.